sites/public/styles/overrides.scss

```diff
@@ -0,0 +1,10 @@
+.ui-icon svg {
```

Comment:
Can we add a comment at the top explaining what this overrides file is for? Like "style overides specific to a jurisdiction" or something?

Reply:
And/or rename the file to that effect

Reply:
Ah, so I think we're going with the assumption that everything under sites/public is Detroit specific at this point. We have other Detroit configurations under sites/public like the ones in [my previous pull request](https://github.com/CityOfDetroit/bloom/pull/5/files).

We can move this to a site specific directory in the future like sites/public-detroit as outlined here: https://github.com/bloom-housing/bloom#adding-a-new-app

I would prefer not to change the file name, because I want to keep the structure consistent with other Bloom implementations. 

Let me know if that's enough, or you still want a comment in the file related to this.

Reply:
Makes sense re the filename!

I think a comment explaining why the overrides exist could still be helpful for future engineers that don't have the context of seeing other bloom implementations.

Without that context, it's not clear that these overrides exist to keep detroit specific styles separate from upstream. (Another common use for overrides files is to override framework code, which is what I think someone new to this repo might assume this file is.)

Reply:
Done!